mlflow/server/js/src/experiment-tracking/components/ExperimentListView.js

```diff
@@ -90,10 +90,19 @@ export class ExperimentListView extends Component {
     this.updateSelectedExperiment('0', '');
   };
 
+  handleCheck = () => (checkedKeys, e) => {
+    if (checkedKeys.length === 0) {
+      // Do not allow un-checking all
+      e.node.checked = true;
+      return;
+    }
+    this.props.history.push(Routes.getCompareExperimentsPageRoute(checkedKeys));
+  };
+
   renderListItem = ({ title, key }) => {
-    const { activeExperimentId } = this.props;
-    const dataTestId =
-      activeExperimentId === key ? 'active-experiment-list-item' : 'experiment-list-item';
+    const { activeExperimentIds } = this.props;
+    const isActive = activeExperimentIds !== undefined && activeExperimentIds.includes(key);
```

Comment:
```suggestion
    const isActive = activeExperimentIds?.includes(key);
```

Reply:
Nifty! I didn't know about this optional chaining operator. Maybe I can also combine it with ?? to make sure isActive is actually a boolean no matter what the value of activeExperimentIds is?